### DISCUSSION TITLE
Expose EmojiAvatar Component in RainbowKit

### DISCUSSION THREAD

**Discussion by misterjame** [rainbow-me/rainbowkit#2266] [Ideas]
I’d like to request that RainbowKit exposes its `EmojiAvatar` component (or a similar API) so developers can use the familiar avatar representation (color + emoji) in their custom implementations. Currently, the docs allow customizing the avatar using the address to map a color, but there’s no easy way to include the emoji, which is a key part of RainbowKit’s visual identity.

Users are already used to the default avatar style, and reimplementing the emoji mapping logic outside of RainbowKit is redundant and error-prone. It would be great if we could use the same color + emoji combination that RainbowKit provides by default.

**Comment by DanielSinclair** [comment]
Will evaluate. You're welcome to drop our implementation into your codebase in the meantime: https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/components/Avatar/emojiAvatarForAddress.ts

**Reply by nikoumba** [reply]
Yes that works! 
Even that I would prefer it to be exported from RainbowKit
